tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
I agree with that. I wouldn't expect failure scenarios to be described in Gherkin. Like the cliché says, the Gherkin scenarios describe a contract with the user, and we hardly want 
```
When I run a command this way
Then Terminus screws up
```
in the contract.